library/extractor/src/main/java/com/google/android/exoplayer2/extractor/mp4/FragmentedMp4Extractor.java

```diff
@@ -470,6 +477,28 @@ private void onLeafAtomRead(LeafAtom leaf, long inputPosition) throws ParserExce
       haveOutputSeekMap = true;
     } else if (leaf.type == Atom.TYPE_emsg) {
       onEmsgLeafAtomRead(leaf.data);
+    } else if (leaf.type == Atom.TYPE_prft) {
+      parsePrft(leaf.data);
+    }
+  }
+
+  private void parsePrft(ParsableByteArray atom) {
+    atom.setPosition(Atom.HEADER_SIZE);
+    int fullAtom = atom.readInt();
+    int version = Atom.parseFullAtomVersion(fullAtom);
+    int flags = Atom.parseFullAtomFlags(fullAtom);
+    String type = "encoder";
+    if ((flags & 24) == 24) {
+      type = "captured";
+    }
+
+    long referenceTrackId = atom.readUnsignedInt();
+    long ntpTimestamp = atom.readLong();
+    long mediaTime = version == 0 ? atom.readUnsignedInt() : atom.readUnsignedLongToLong();
+
+    if (prftHandler != null) {
+      Log.i(TAG, "PRFT decoded");
```

Comment:
Probably info level is too high

Reply:
Sorry that line should've been removed. Will fix that asap.